lib/norm/core/selection.ex

```diff
@@ -49,7 +49,7 @@ defmodule Norm.Core.Selection do
   defp validate_selectors!([]), do: true
   defp validate_selectors!([{_key, inner} | rest]), do: validate_selectors!(inner) and validate_selectors!(rest)
   defp validate_selectors!([_key | rest]), do: validate_selectors!(rest)
-  defp validate_selectors!(other), do: raise ArgumentError, "select expects a list of keys but received: #{inspect other}"
+  defp validate_selectors!(other), do: raise(ArgumentError, "select expects a list of keys but received: #{inspect other}")
```

Comment:
Fixes the following compilation warning:

<img width="1727" alt="Screenshot 2024-05-29 at 1 19 53 PM" src="https://github.com/elixir-toniq/norm/assets/34700/3af02d34-3c2e-4edc-b2b8-460c63c71b79">